lib/ansible/modules/identity/keycloak/keycloak_client.py

```diff
@@ -530,69 +562,70 @@
     web_origins:
       - https://www.example.com/*
     not_before: 1507825725
-    bearer_only: False
-    consent_required: False
-    standard_flow_enabled: True
-    implicit_flow_enabled: False
-    direct_access_grants_enabled: False
-    service_accounts_enabled: False
-    authorization_services_enabled: False
-    public_client: False
-    frontchannel_logout: False
+    bearer_only: no
+    consent_required: no
+    standard_flow_enabled: yes
+    implicit_flow_enabled: no
+    direct_access_grants_enabled: no
+    service_accounts_enabled: no
+    authorization_services_enabled: no
+    public_client: no
+    frontchannel_logout: no
     protocol: openid-connect
-    full_scope_allowed: false
+    full_scope_allowed: no
     node_re_registration_timeout: -1
     client_template: test
-    use_template_config: False
-    use_template_scope: false
+    use_template_config: no
+    use_template_scope: no
     use_template_mappers: no
     registered_nodes:
       node01.example.com: 1507828202
     registration_access_token: eyJWT_TOKEN
-    surrogate_auth_required: false
+    surrogate_auth_required: no
     default_roles:
       - test01
       - test02
     protocol_mappers:
       - config:
-          access.token.claim: True
-          claim.name: "family_name"
-          id.token.claim: True
+          access.token.claim: yes
+          claim.name: family_name
+          id.token.claim: yes
           jsonType.label: String
           user.attribute: lastName
-          userinfo.token.claim: True
-        consentRequired: True
-        consentText: "${familyName}"
+          userinfo.token.claim: yes
+        consentRequired: yes
+        consentText: '{{ familyName }}'
```

Comment:
This is not the same, ${familyName} is the actual string literal intended to be given to the API here, not a JINJA replacement or such.

Reply:
Ok, I thought this was old (very old even) Ansible syntax :-)

Reply:
Fixed.